src/spirc.rs

```diff
@@ -57,7 +65,7 @@ impl SpircManager {
             name: name,
             ident: ident,
             device_type: 5,
-            can_play: true,
+            can_play: player_is_some,
```

Comment:
Grayed out, saying "Unavailable for listening on". I removed the capability, but it didn't make any difference. We can keep it though, as it's probably the right thing to do.

Are you sure this isn't the intended behaviour? Are there devices you can buy which doesn't show up in the device list?